cmake/gau2gridConfig.cmake.in

```diff
@@ -49,6 +51,12 @@ set (_valid_components
     shared
 )
 
+if(${SPHERICAL_ORDER} STREQUAL "cca")
```

Comment:
Will "CCA" match as well?

Reply:
no. cmake is rotten at cases. `OR` if you want to be generous to the user. I've done all components in the psi eco to lowercase, so I'd prefer "OR" to outright uppercase.

Reply:
not w/o writing some checking code. latest commit to main CM constrains this. I haven't looked for where `SPHERICAL_ORDER` is actually used.

Reply:
looks like you weren't using mixed or uppercase in code anyways. so should be ok.